blocks/shared-styles/_children/promo-label/index.jsx

```diff
@@ -38,7 +24,7 @@ const LabelBoxSmall = styled.div`
   align-items: center;
   padding: 8px;
   background-color: ${(props) => props.primaryColor};
-  border: none;
+  border: 0;
```

Comment:
this was a deliberate change? I vaguely remember hearing about some substantive difference between border none and 0 https://github.com/WPMedia/fusion-news-theme-blocks/pull/778